src/wallet/wallet.cpp

```diff
@@ -4135,3 +4139,110 @@ bool CWalletTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState&
     fInMempool = ret;
     return ret;
 }
+
+static const std::string OUTPUT_TYPE_STRING_LEGACY = "legacy";
+static const std::string OUTPUT_TYPE_STRING_P2SH = "p2sh";
+static const std::string OUTPUT_TYPE_STRING_BECH32 = "bech32";
+
+OutputType ParseOutputType(const std::string& type, OutputType default_type)
+{
+    if (type == "") {
```

Comment:
I would suggest to use `type.empty()` instead, no need to call string comparison fn. Generated code is much better.

Reply:
Fixed.